cmd/antrea-agent/agent.go

```diff
@@ -922,6 +923,11 @@ func run(o *Options) error {
 		go flowExporter.Run(stopCh)
 	}
 
+	// Start the node latency monitor.
+	// TODO: Set interval and timeout in config.
+	nodeLatencyMonitor := monitortool.NewNodeLatencyMonitor(nodeInformer, time.Second*10, time.Second*5)
+	go nodeLatencyMonitor.Run(stopCh)
```

Comment:
There should be a FeatureGate which disables this feature by default while it is still in development stage: https://github.com/antrea-io/antrea/blob/main/pkg/features/antrea_features.go (Alpha)

The feature should also be configurable through the antrea-agent config (https://github.com/antrea-io/antrea/blob/main/pkg/config/agent/config.go)

Reply:
Thanks, I will update it.